packages/ui-common/src/TxQueueContext.tsx

```diff
@@ -193,8 +195,8 @@ export function TxQueueContextProvider (props: Props) {
   /**
    * Clear the txQueue.
    */
-  const clear = () => {
-    let msg: string[] = [];
+  const clear = (): void => {
+    const msg: string[] = [];
```

Comment:
this is not a const

Reply:
From a JS point of view, it's a const, we don't modify the reference

Reply:
ah right, I forgot about that, nevermind then